src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +396,50 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Clear current language filter vector, lest all entries are shown twice
+	addon_manager::language_filter_types_.clear();
+	addon_manager::language_filter_types_.push_back(std::pair<int, std::string>(0, "en_GB"));
+
+	// Prepare shown languages, source all available languages from the addons themselves
+	std::vector<std::string> languages_available;
+	for(const auto& a : addons_) {
+		for (const auto& b : a.second.locales) {
+			if (!(b.rfind("en_GB", 0) == 0)) {
```

Comment:
`en_GB` should not be hardcoded. If anything, it should be your current game language selection.

Reply:
:ok_hand: 
I now changed it to no particular ordering now en_GB hardcode was for programming convenience in get_lang_filter_visibility(), see other discussion